shortstories/views.py

```diff
@@ -3,6 +3,8 @@
 from .forms import ShortStoryForm
 from  django import forms
 from  django.urls import reverse
+from  django.utils.decorators import method_decorator
```

Comment:
Could you please tell me why you needed to import this line?

Reply:
I tested one code,it had decorator and when I run, it gave me error that  decorator  is not defined, so I imported that, but I changed that code and did not need decorator anymore but forgot to remove it's import.